docs/api/covidcast.md

```diff
@@ -14,45 +14,136 @@ General topics not specific to any particular data source are discussed in the
-Facebook.  These surveys are voluntary, and no individual survey responses are
-shared back to Facebook. Using this survey data, we estimate the percentage of
-people in a given location, on a given day that have CLI (covid-like illness =
-fever, along with cough, or shortness of breath, or difficulty breathing), and
-separately, that have ILI (influenza-like illness = fever, along with cough or
-sore throat).
-* `google-survey`: Data signal based on Google-run symptom surveys, through
-their Opinions Reward app, and similar applications.  These surveys are again
-voluntary.  They are just one question long, and ask "Do you know someone in
-your community who is sick (fever, along with cough, or shortness of breath, or
-difficulty breathing) right now?"  Using this survey data, we estimate the
+Delphi's COVID-19 Surveillance Streams data includes the following data sources.
+Data from these sources is expected to be updated daily. You can use the
+[`covidcast_meta`](covidcast_meta.md) endpoint to get summary information about
+the ranges of the different attributes for the different data sources currently
+in the data.
+
+#### `fb-survey`
+
+This indicator estimates the percentage of people who have a COVID-like illness
+(fever, along with cough, or shortness of breath, or difficulty breathing) or
+influenza-like illness (fever, along with cough or sore throat), based on
+symptom surveys run by Carnegie Mellon. Facebook directs a random sample of its
+users to these surveys, which are voluntary. Individual survey responses are
+held by CMU and are shareable with other health researchers under a data use
+agreement. No individual survey responses are shared back to Facebook.
+
+| Signal | Description |
+| --- | --- |
+| `raw_cli` | Estimated fraction of people with COVID-like illness, with no smoothing or survey weighting |
+| `raw_ili` | Estimated fraction of people with influenza-like illness, with no smoothing or survey weighting |
+| `raw_wcli` | Estimated fraction of people with COVID-like illness; adjusted using survey weights |
+| `raw_wili` | Estimated fraction of people with influenza-like illness; adjusted using survey weights |
+
+The survey weights, provided by Facebook, are intended to make the sample
+representative of the US population, according to the state, age, and gender of
+the US population from the 2018 Census March Supplement.
+
+Along with the `raw_` signals, there are additional signals with names beginning
+with `smoothed_`. These estimate the same quantities as the above signals, but
+smoothed in time to reduce day-to-day sampling noise.
+
+#### `google-survey`
+
+This indicator estimates the percentage of people who know someone in their
+community with a COVID-like illness (fever, along with cough, or shortness of
+breath, or difficulty breathing). The data is based on Google-run symptom
+surveys, through publisher websites, their Opinions Reward app, and similar
+applications. These surveys are voluntary.
```

Comment:
note to self: these are "voluntary" only in the sense that you can opt to do a different survey